### DISCUSSION TITLE
match accented characters when filtering by unaccented characters

### DISCUSSION THREAD

**Discussion by olets** [brianvoe/slim-select#437] [Ideas]
Native `select` will match accented characters when you type its unaccented form.

Example: open the select at https://codepen.io/henry/pen/VwBojdV and type `e`. See that `é` is focused.

slim-select doesn't emulate this. To get `é` you have to type accented `é`.

Example: type `e` into the slim-select filter at https://codepen.io/henry/pen/dyjxXJq. See that the only match is `e`.

My feature request includes matching not only at the start of the search string. In the second example, it'd be great if `e` matched `e`, `é`, and `abc César`.

**Comment by brianvoe** [comment]
https://slimselectjs.com/events#searchFilter
You can always set your own filter if you would like. 
If you have a suggestion for a better default please submit a pr.

Thanks!

**Reply by olets** [reply]
GitHub isn't showing me the "mark as answer" button. Maybe it doesn't for things tagged "Ideas"? So: thanks, that's the answer ✅

**Reply by olets** [reply]
This popped up in my notifications. Don't know why, but to close that tangent in case you're interested: https://github.com/orgs/community/discussions/70108#discussioncomment-7296262
